numpy/core/_add_newdocs.py

```diff
@@ -1564,6 +1564,11 @@
         array_function_like_doc,
     ))
 
+add_newdoc('numpy.core.multiarray', 'from_dlpack',
+    """
+    Create a NumPy array from a DLPack struct.
```

Comment:
```suggestion
    from_dlpack(obj, /)

    Create a NumPy array from an object implementing the ``__dlpack__`` protocol.
```
Three things:
1. The signature was missing.
2. The docstring previously claimed to construct an array from a DLPack struct. 
  From what I understand this is not correct though, as the function takes any object (correctly) implementing the ``__dlpack__`` protocol (which in turn does return a PyCapsule pointing to the DLPack struct in question).
3. It might be useful to add a See Also section with a link to the API-specs of ``__dlpack__``.

Reply:
Come to think of it, the docstrings of ``ndarray.__dlpack__`` and ``ndarray.__dlpack_device__`` are also missing.